Moller-Plesset/MP2_Hessian.py

```diff
@@ -0,0 +1,776 @@
+        #
+        Hes["R"][r][c] += 0.5 * 1.0 * np.einsum('pqrs,tp,tqrs->', dPpqrs[key1], U[key2], npERI, optimize=True)
+        Hes["R"][r][c] += 0.5 * 1.0 * np.einsum('pqrs,tq,ptrs->', dPpqrs[key1], U[key2], npERI, optimize=True)
+        Hes["R"][r][c] += 0.5 * 1.0 * np.einsum('pqrs,tr,pqts->', dPpqrs[key1], U[key2], npERI, optimize=True)
+        Hes["R"][r][c] += 0.5 * 1.0 * np.einsum('pqrs,ts,pqrt->', dPpqrs[key1], U[key2], npERI, optimize=True)
+        Hes["R"][r][c] += 0.5 * 1.0 * np.einsum('pqrs,tp,tqrs->', dPpqrs[key2], U[key1], npERI, optimize=True)
+        Hes["R"][r][c] += 0.5 * 1.0 * np.einsum('pqrs,tq,ptrs->', dPpqrs[key2], U[key1], npERI, optimize=True)
+        Hes["R"][r][c] += 0.5 * 1.0 * np.einsum('pqrs,tr,pqts->', dPpqrs[key2], U[key1], npERI, optimize=True)
+        Hes["R"][r][c] += 0.5 * 1.0 * np.einsum('pqrs,ts,pqrt->', dPpqrs[key2], U[key1], npERI, optimize=True)
+
+
+# Build Total Hessian
+for key in Hes:
+    Hessian += Hes[key]
+
+# Symmetrize Hessian
+Hessian = (Hessian + Hessian.T)/2
+
+Mat = psi4.core.Matrix.from_array(Hessian)
+Mat.name = " TOTAL HESSIAN"
+Mat.print_out()
+
+H_psi4 = psi4.core.Matrix.from_list([
+[-3.6130997071,        0.0000000000,       -0.0000000000,        1.8065498535,       -0.0000000000,       -0.0000000000,        1.8065498535,        0.0000000000,        0.0000000000],
+[ 0.0000000000,        8.4484978101,        0.0000000000,       -0.0000000000,       -4.2242489050,       -4.7117763859,        0.0000000000,       -4.2242489050,        4.7117763859],
+[-0.0000000000,        0.0000000000,        3.7558758529,       -0.0000000000,       -4.3809411905,       -1.8779379264,        0.0000000000,        4.3809411905,       -1.8779379264],
+[ 1.8065498535,       -0.0000000000,       -0.0000000000,       -1.8756582110,        0.0000000000,        0.0000000000,        0.0691083574,        0.0000000000,        0.0000000000],
+[-0.0000000000,       -4.2242489050,       -4.3809411906,        0.0000000000,        4.3722044693,        4.5463587882,       -0.0000000000,       -0.1479555642,       -0.1654175976],
+[-0.0000000000,       -4.7117763860,       -1.8779379264,        0.0000000000,        4.5463587882,        1.8072072616,        0.0000000000,        0.1654175977,        0.0707306648],
+[ 1.8065498535,        0.0000000000,        0.0000000000,        0.0691083574,       -0.0000000000,        0.0000000000,       -1.8756582110,       -0.0000000000,       -0.0000000000],
+[ 0.0000000000,       -4.2242489050,        4.3809411906,        0.0000000000,       -0.1479555642,        0.1654175976,       -0.0000000000,        4.3722044693,       -4.5463587882],
+[ 0.0000000000,        4.7117763860,       -1.8779379264,        0.0000000000,       -0.1654175977,        0.0707306648,       -0.0000000000,       -4.5463587882,        1.8072072616]
+])
+H_python_mat = psi4.core.Matrix.from_array(Hessian)
+psi4.compare_matrices(H_psi4, H_python_mat, 9, "RHF-HESSIAN-TEST")
```

Comment:
`H_psi4` is from CFOUR? Good to add that `ZMAT` to the dir.

`RHF-HESSIAN-TEST` --> `RMP2-HESSIAN-TEST`?